src/style.css

```diff
@@ -0,0 +1,128 @@
+@import "./stars.css";
+
+.page {
+  font-family: Roboto, Helvetica, sans-serif;
+}
+
+.content {
+  --big_text_color: #060b35;
+  --small_text_color: #616070;
+
+  margin: 0;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.card {
```

Comment:
Move each BEM block to the separate file following next approach <BlockName>.css